packages/environment-check/src/checks/install.ts

```diff
@@ -0,0 +1,64 @@
+import { existsSync, promises } from 'fs';
+import { t } from '../i18n';
+import { join } from 'path';
+import { spawnCommand, npmCommand } from '../command';
+
+/**
+ * Reads the list of extensions installed in vscode.
+ *
+ * @returns list of extension ids
+ */
+export const getInstalledExtensions = async (): Promise<{ [id: string]: { version: string } }> => {
+    const output = await spawnCommand('code', ['--list-extensions', '--show-versions']);
+    const versions = output
+        .split('\n')
+        .filter((ext) => ext.startsWith('SAP'))
+        .reduce((returnObject, current) => {
+            const [id, version] = current.split('@');
+            returnObject[id] = {
+                version
+            };
+            return returnObject;
+        }, {});
+    return versions;
+};
+
+/**
+ * Read the version of the cloud foundry CLI.
+ *
+ * @param module - module id
+ * @returns version
+ */
+export const getCFCliToolVersion = async (module: string): Promise<string> => {
```

Comment:
The name suggests that this function returns the version of the "Clound Foundry CLI (cf)", but you still need to pass parts of the CLI command into it. I'd say, either this function returns the version of cf and doesn't require the consumer to know about the command name (`cf`), or we rename this function to indicate, that you can pass any command line command and it will return the version (as long as argument -v returns the version. I'm in favor of the first and made the suggestions accordingly.

```suggestion
export const getCFCliToolVersion = async (): Promise<string> => {
```

Reply:
Updated as suggested